From 5ef7b1c08af3735f46d62393ead00beb6944461b Mon Sep 17 00:00:00 2001
From: David Engel <dengel@mythtv.org>
Date: Fri, 6 Jun 2014 14:10:57 -0500
Subject: [PATCH 08/13] Retry to fix corner cases when stopping in-progress
 recordings.

This change retains a small part of the previous 8116fde7 commit.  The
rest of the previous commit was obviated by the recstartts part of
commit e8a99d45.  The ideal fix will ultimately use recordedid instead
of chanid/recstartts, but that will come later.
---
 mythtv/programs/mythfrontend/schedulecommon.cpp |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/mythtv/programs/mythfrontend/schedulecommon.cpp b/mythtv/programs/mythfrontend/schedulecommon.cpp
index 65f140b..11b83de 100644
--- a/mythtv/programs/mythfrontend/schedulecommon.cpp
+++ b/mythtv/programs/mythfrontend/schedulecommon.cpp
@@ -452,10 +452,7 @@ void ScheduleCommon::customEvent(QEvent *event)
             }
             else if (resulttext == tr("Stop this recording"))
             {
-                ProgramInfo pginfo(
-                    recInfo.GetChanID(), recInfo.GetRecordingStartTime());
-                if (pginfo.GetChanID())
-                    RemoteStopRecording(&pginfo);
+                RemoteStopRecording(&recInfo);
             }
             else if (resulttext == tr("Modify recording options") ||
                      resulttext == tr("Add override rule"))
-- 
1.7.10.2

